src/pilot/templates/dellnfv/createaggregate.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2016-2018 Dell Inc. or its subsidiaries.
```

Comment:
Can you change the copyright date to 2018?

Reply:
Fixed